members/admin/__init__.py

```diff
@@ -1,777 +1,50 @@
-            message_bit = "1 familie"
-        else:
-            message_bit = "%s familier" % queryset.count()
-        self.message_user(request, "%s fik fik tilsendt link e-mail." % message_bit)
-
-    resend_link_email.short_description = "Gensend link e-mail"
-
-    # Only view familys related to users department (via participant, waitinglist & invites)
-    def get_queryset(self, request):
-        qs = super(FamilyAdmin, self).get_queryset(request)
-        if request.user.is_superuser:
-            return qs
-        departments = Department.objects.filter(
-            adminuserinformation__user=request.user
-        ).values("id")
-        return qs.filter(
-            Q(person__member__activityparticipant__activity__department__in=departments)
-            | Q(person__waitinglist__department__in=departments)
-            | Q(person__activityinvite__activity__department__in=departments)
-        ).distinct()
-
-
-admin.site.register(Family, FamilyAdmin)
-
-
-class ParticipantPaymentListFilter(admin.SimpleListFilter):
-    # Title shown in filter view
-    title = "Betaling"
-
-    # Parameter for the filter that will be used in the URL query.
-    parameter_name = "payment_list"
-
-    def lookups(self, request, model_admin):
-        """
-        Returns a list of tuples. The first element in each
-        tuple is the coded value for the option that will
-        appear in the URL query. The second element is the
-        human-readable name for the option that will appear
-        in the right sidebar.
-        """
-
-        activitys = [
-            ("pending", "Afventende"),
-            ("rejected", "Afvist"),
-            ("ok", "Betalt"),
-            ("none", "Ikke betalt"),
-            ("confirmed", "Hævet"),
-        ]
-        return activitys
-
-    def queryset(self, request, queryset):
-        """
-        Returns the filtered queryset based on the value
-        provided in the query string and retrievable via
-        `self.value()`.
-        """
-        # Compare the requested value (either '80s' or '90s')
-        # to decide how to filter the queryset.
-
-        if self.value() == "none":
-            return queryset.filter(payment__isnull=True)
-        elif self.value() == "ok":
-            return queryset.filter(
-                payment__isnull=False, payment__accepted_at__isnull=False
-            )
-        elif self.value() == "confirmed":
-            return queryset.filter(
-                payment__isnull=False, payment__confirmed_at__isnull=False
-            )
-        elif self.value() == "pending":
-            return queryset.filter(
-                payment__isnull=False, payment__confirmed_at__isnull=True
-            )
-        elif self.value() == "rejected":
-            return queryset.filter(
-                payment__isnull=False, payment__rejected_at__isnull=False
-            )
-
-
-class ActivityParticipantListOldYearsFilter(admin.SimpleListFilter):
```

Comment:
Dette `ActivityParticipantListOldYearsFilter` ser ud til at være slettet, kan i hvert fald ikke finde i andre filer i dette PR. Er det ok?

Reply:
Blev tilføjet for nogle måneder siden i https://github.com/CodingPirates/forenings_medlemmer/pull/747. @mhewel, formoder vi ønsker at beholde dette filter?